io/Cargo.toml

```
@@ -1,11 +1,11 @@
 [package]
 name = "staking-io"
-version = "0.1.0"
+version = "0.1.1"
```

And add an entry to CHANGELOG.
```suggestion
version = "0.1.2"
```